### ISSUE TITLE
I want to ask if there are any errors in my training parameter settings? Why is the performance of my mode not very good?

### ISSUE THREAD

**Issue by silvester666** [VDIGPKU/QGCN#3]
python src/main.py --dataloader jpeg_qm --size 64 --batchSize 256 --quality [10,50] --colorMode RGB --net qmar --n_resblock 64 --loss L1 --cuda --threads 4 --gpus [0] --savedir QCN --nEpochs 50 --imgGTRoot src/data/DIV2K_train_HR/

**Comment by silvester666** [comment]
Do the downloaded raw images need to be processed before being used as the training set?
